Source/Core/VideoCommon/AVIDump.cpp

```diff
@@ -66,7 +66,7 @@ bool AVIDump::Start(HWND hWnd, int w, int h)
 	if (SConfig::GetInstance().m_SYSCONF->GetData<u8>("IPL.E60"))
 		s_frame_rate = 60; // always 60, for either pal60 or ntsc
 	else
-		s_frame_rate = VideoInterface::TargetRefreshRate; // 50 or 60, depending on region
+		s_frame_rate = (int)(VideoInterface::TargetRefreshRate + 0.5f);
```

Comment:
The proper fix is treating the framerate as a rational number rather than an integer (AVI stores the framerate as a ratio of dwRate/dwScale).